internal/config.go

```diff
@@ -331,10 +332,15 @@ func (config *Config) processToml(tomlPath string) error {
 		networkArchiveUrls = network.TestNetworkhistoryArchiveURLs
 		config.SerializedCaptiveCoreToml = ledgerbackend.TestnetDefaultConfig
 
+	case Futurenet:
```

Comment:
could add test coverage to assert this outcome, there is an existing unit test [TestValidCaptiveCorePreconfiguredNetwork](https://github.com/stellar/stellar-galexie/blob/add-futurenet-network-config/internal/config_test.go#L108) which asserts `pubnet` works for reference.